app/src/main/java/com/mapzen/erasermap/CommonModule.java

```diff
@@ -26,7 +27,9 @@
         return new Bus();
     }
 
-    @Provides RouteEngine provideRouteEngine() {
+    @Provides @Singleton ManifestModel provideApiKeys() { return new ManifestModel(); }
```

Comment:
Typically Dagger provider methods will mirror name of return type. For example `provideManifestModel()`.